packages/expo-cli/package.json

```diff
@@ -15,8 +15,8 @@
     "prepare": "yarn run clean && yarn run build",
     "clean": "rimraf build ./tsconfig.tsbuildinfo",
     "preversion": "node ./scripts/preversion.js",
-    "test": "jest",
-    "test:watch": "jest --watch"
+    "test": "jest --config jest/config.json",
+    "test:watch": "jest --config jest/config.json --watch"
```

Comment:
FYI https://github.com/expo/expo-cli/pull/2117 will put the config in `packages/expo-cli/jest.config.js`. We're going to standardize the config location across packages, and I think we'll want to use that default location for that, so please use `jest.config.js` for the config to avoid unnecessary merge conflicts. 😄

Reply:
Uh, I didn't know that. I'll close the other PR (https://github.com/expo/expo-cli/pull/2147/files).

Reply:
Haha, I already beat you to it 😄